src/inflation_utils.py

```diff
@@ -0,0 +1,47 @@
+# Inflation Data is currently only from Canada
+
+import numpy as np 
+import json
+from requests import post
+from datetime import datetime
+
+class InflationUtils:
+	def __init__(self, start_date, end_date = str(datetime.date(datetime.now()))):
+		self._start_date = start_date
+		self._end_date = end_date
+		self.data = self.get_inflation_series()
+
+	def get_inflation_series(self):
+		url = 'https://www150.statcan.gc.ca/t1/wds/rest/getBulkVectorDataByRange'
```

Comment:
Usually, all constants are stored in the __init__ method that way its easier to look at them all together and change them in 1 place